partner_firstname/models/res_partner.py

```diff
@@ -192,11 +192,23 @@ def _get_inverse_name(self, name, is_company=False):
 
     @api.multi
     def _inverse_name(self):
-        """Try to revert the effect of :meth:`._compute_name`."""
+        """
+            Try to revert the effect of :meth:`._compute_name`.
+            Generate one write if values are really updated
+        """
         for record in self:
             parts = record._get_inverse_name(record.name, record.is_company)
-            record.lastname = parts['lastname']
-            record.firstname = parts['firstname']
+            to_update = {}
```

Comment:
use values instead of to_update, as this is also the actual argument name in to_update.

Reply:
Didn't catch what you mean, but done